test/integration/serverless/serverless.yml

```diff
@@ -86,24 +81,34 @@ functions:
     layers:
       - { Ref: RecorderExtensionLambdaLayer }
       - { Ref: DatadogExtensionIntegrationTestLambdaLayer }
-    environment:
-      DD_TAGS: tagA:valueA tagB:valueB
-      DD_EXTRA_TAGS: tagC:valueC tagD:valueD
 
   metric-csharp:
     runtime: dotnetcore3.1
     handler: CsharpHandlers::AwsDotnetCsharp.Handler::Hello
     package:
       individually: true
       artifact: src/csharp-tests/bin/Release/netcoreapp3.1/handler.zip
+    layers:
+      - { Ref: RecorderExtensionLambdaLayer }
+      - { Ref: DatadogExtensionIntegrationTestLambdaLayer }
+      # TODO: Replace with production layer when released
+      - arn:aws:lambda:sa-east-1:601427279990:layer:poc-maxday-dotnet:130
```

Comment:
nit : you can define this layer only once at the top of this file (after `datadogExtensionIntegrationTest`) then reference it using ` - { Ref: xxxLayer }`
By doing this we'll need to update the layer version at only one place